include/common_structs.h

```diff
@@ -1506,13 +1506,13 @@ typedef struct PushBlockGrid {
 typedef struct ItemEntityPhysicsData {
     /* 0x00 */ f32 verticalVelocity;
     /* 0x04 */ f32 gravity; /* 2 = normal, 1 = low gravity, higher values never 'settle' */
-    /* 0x08 */ f32 unk_08;
+    /* 0x08 */ f32 collisionRadius;
     /* 0x0C */ f32 constVelocity;
     /* 0x10 */ f32 velx;
     /* 0x14 */ f32 velz;
     /* 0x18 */ f32 moveAngle;
     /* 0x1C */ s32 timeLeft;
-    /* 0x20 */ s32 unk_20;
+    /* 0x20 */ s32 useSimplePhysics;
```

Comment:
The way this is named, it seems like a bool

Reply:
Good catch